configure.ac

```diff
@@ -205,6 +205,12 @@ AS_IF([test -z "$cargo"], [AC_MSG_ERROR([cargo is required for --enable-rust])])
 AC_PATH_PROG([rustc], [rustc])
 AS_IF([test -z "$rustc"], [AC_MSG_ERROR([rustc is required for --enable-rust])])
 
+AS_IF([test -f "${srcdir}"/rpmostree-rust.h],
```

Comment:
Hmm, so if I understand this right, this can cause issues with `rpmostree-rust.h` not being regenerated if I happen to rerun `configure`, right? Can we also base this off of e.g. `test -e "${srcdir}"/.git` or something? Hmm, except that will still fail in the `autosetup -Sgit` case. Maybe simplest a special stamp file we add to the archive from `make -f Makefile.dist-packaging dist-snapshot`?

Reply:
> Hmm, so if I understand this right, this can cause issues with rpmostree-rust.h not being regenerated if I happen to rerun configure, right?

Oooh.  Good call.

> Maybe simplest a special stamp file we add to the archive from make -f Makefile.dist-packaging dist-snapshot

Fixed :arrow_down: